src/app/container/descriptors/descriptors.component.html

```diff
@@ -56,9 +56,8 @@
       </mat-toolbar>
       <app-code-editor [(content)]="content" [filepath]="filePath"></app-code-editor>
     </div>
-    <alert type="warning" *ngIf="nullDescriptors">
-      <mat-icon>warning</mat-icon>
-      &nbsp;A Descriptor File associated with this Docker container could not be found.
-    </alert>
+    <mat-card class="alert alert-warning" role="alert" *ngIf="nullDescriptors">
+      <mat-icon>warning</mat-icon> A Descriptor File associated with this Docker container could not be found.
```

Comment:
I realize it was like this already, but I think `Descriptor File` should be `descriptor file`.

Reply:
I think i just did it for emphasis, but there is probably a better way to do that. Maybe similar to what github does (like in your comment)